src/app/features/login/login.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <div class="col-xs-12 col-md-6 col-md-offset-3">
```

Comment:
No need to use Bootstrap. 
Grid system in Angular material:
https://material.angular.io/components/grid-list/overview